packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -345,15 +368,17 @@ export class TokenRatesController extends BaseController<
   /**
    * Fetches a pairs of token address and native currency.
    *
-   * @param chainSlug - Chain string identifier.
-   * @param vsCurrency - Query according to tokens in tokenList and native currency.
+   * @param chainSlug - The chain string identifier.
+   * @param vsCurrency - The currency used to generate pairs against the tokens.
+   * @param tokenAddresses - The addresses for the token contracts.
    * @returns The exchange rates for the given pairs.
    */
   async fetchExchangeRate(
```

Comment:
Also can this be made private? Helps to keep track of what changes should be considered and breaking and how to think about the method signatures

Reply:
This one is stubbed out a few times but we should be able to test the public methods without referring to it. I know its kindof annoying but it will make our tests better and this controller better if we make it private.

Reply:
Ok lets make this private in a subsequent PR per @mcmire 's[ suggestion](https://consensys.slack.com/archives/C063W9EE484/p1700166880644369)